blockchain/types/anchoring_data.go

```diff
@@ -22,6 +22,10 @@ import (
 	"math/big"
 )
 
+const (
+	AnchoringDataType0 uint8 = 0
+)
+
```

Comment:
How about enum and iota?

Reply:
Simple int literal is used to explicitly control the version number. Since the type is written to the block, I think that iota's implicit numbering might be dangerous.